reference/src/layout/arrays-and-slices.md

```diff
@@ -1,5 +1,10 @@
 # Layout of Rust array types and slices
 
+## Editor's Note
+
+Issue #176 clarified that as of this writing, Rust does not distinguish between size and stride, ie
+they are always equal.
```

Comment:
At the very least, the "Guaranteeing stride == size" section should probably be removed? And generally the entire file should be updated to assume size == stride I think -- right now different parts of the document seem to contradict each other.